.circleci/cimodel/data/caffe2_build_definitions.py

```diff
@@ -63,6 +73,8 @@ def gen_docker_image(self):
         lang_substitutions = {
             "onnx_py2": "py2",
             "onnx_py3.6": "py3.6",
+            "onnx_py3.6_part1": "py3.6",
```

Comment:
Can we change part1 => main, part2 => ort?

Reply:
So my argument was that both jobs are running ORT tests. I've split the ORT test by opset <=9 or >9 in both jobs. Do you think it would make sense to name those as main and ort?

Reply:
Yes, that's the direction to move. 

How about landing this first and doing it in another PR?